resources/views/components/task-form.blade.php

```diff
@@ -0,0 +1,198 @@
+<div class="form-container">
+    <h2 class="mb-4">Task Management</h2>
+    <form id="taskForm" action="{{ route('tasks') }}" method="POST">
+        @csrf
+
+        <div class="form-group">
+            <label for="action">Action</label>
+            <select name="action" id="action" class="form-control" required
+                    onchange="document.getElementById('taskSelectGroup').style.display = this.value === 'update' ? 'block' : 'none'">
+                <option value="create">Create New Task</option>
+                <option value="update">Update Task</option>
+            </select>
+        </div>
+
+        <div id="taskSelectGroup" class="form-group" style="display: none;">
+            <label for="task_id">Select Task</label>
+            <select name="task_id" id="task_id" class="form-control" onchange="populateTaskFields(this.value)">
+                <option value="">Select Task</option>
+            </select>
+        </div>
+
+        <div class="form-group">
+            <label for="title">Title</label>
+            <input type="text" name="title" class="form-control" required>
+        </div>
+
+        <div class="form-group">
+            <label for="description">Description</label>
+            <textarea name="description" class="form-control" rows="3" required></textarea>
+        </div>
+
+        <div class="form-group">
+            <label for="status">Status</label>
+            <select name="status" class="form-control">
+                <option value="pending">Pending</option>
+                <option value="in_progress">In Progress</option>
+                <option value="completed">Completed</option>
+            </select>
+        </div>
+
+        <div class="form-group">
+            <label for="assigned_to">Assign to Employee</label>
+            <select name="employee_id" class="form-control" required onclick="loadEmployees(this)">
+                <option value="">Select Employee</option>
+            </select>
+        </div>
+
+        <button type="submit" class="btn btn-primary">Submit</button>
+    </form>
+</div>
+
+<script>
+    let employeesLoaded = false;
+    let tasksLoaded = false;
+
+    function loadEmployees(select) {
+    return new Promise((resolve, reject) => {
+        if (employeesLoaded) {
+            resolve();
+            return;
+        }
+
+        select.innerHTML = '<option value="">Select Employee</option>';
+
+        fetch('/employees', {
```

Comment:
```suggestion
        fetch('{{ route('employees.index') }}', {
```

Avoid hardcoding API routes. Using `route()` ensures consistency and prevents issues if endpoints change.

Reply:
Thanks about the comment, also the front code was provided by the challenge.